models/trigger.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const Joi = require('joi');
+const Job = require('../config/job');
+const Regex = require('../config/regex');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .example(12345),
+
+    pipelineId: Joi
+        .number().integer().positive()
+        .example(56789),
+
+    jobName: Job.jobname
+        .example('component'),
+
+    trigger: Joi
```

Comment:
`trigger` is a reserved keyword in a lot of SQL database. should probably user something else as the column name

Reply:
Changed the column name to `start` based on your suggestion :)